wunderio/next4drupal-project
# NEX-16 Smart language switcher.

Smart language switcher

*Changes proposed in this PR:*
- Path translations fetched and passed to Lang Switcher component via context

*How to test:*
1. Create an Article or two and translate to Fi, Swe, En.
2. Go in localhost:3000/fi and go to your Fi article.
3. Switch langs in Lang switcher menu and make sure correct pages are rendered and no 404 happen
4. Confirm everything works for homepage
5. Confirm lang switching works if you start from English article first.
